rust-client/Cargo.toml

```diff
@@ -1,7 +1,10 @@
 [package]
 name = "rust-client"
```

Comment:
Leaving name as is for now. Let's change it once we no longer depend on a git reference and thus are ready to publish.